src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -154,11 +163,16 @@ func (df *dualFetcher) RoundTrip(req *http.Request) (*http.Response, error) {
 // header to specify the fronted URL to use.
 func (df *dualFetcher) do(req *http.Request, chainedFunc func(*http.Request) (*http.Response, error), ddfFunc func(*http.Request) (*http.Response, error)) (*http.Response, error) {
 	log.Debugf("Using dual fronter")
-	frontedURL := req.Header.Get("Lantern-Fronted-URL")
-	req.Header.Del("Lantern-Fronted-URL")
+
+	op := ops.Enter("dualfetcher").Request(req)
+	defer op.Exit()
+
+	parallel := df.cf.parallel
+	frontedURL := req.Header.Get(lanternFrontedURL)
+	req.Header.Del(lanternFrontedURL)
 
 	if frontedURL == "" {
-		return nil, errors.New("Callers MUST specify the fronted URL in the Lantern-Fronted-URL header")
+		return nil, op.Error(errors.New("Callers MUST specify the fronted URL in the Lantern-Fronted-URL header"))
```

Comment:
`op.Enter()` will change current context right? `errors.New()` also takes current context into consideration. IIUC `op.Error()` here is unnecessary.

I do prefer to avoid any implicit correlation between `errors` and `ops`, but if we can't avoid it, we'd better disallow explicit link between them to avoid confusing.